src/main/resources/com/github/maven_nar/aol.properties

```diff
@@ -127,9 +127,7 @@ amd64.Windows.msvc.res.excludes=
 
 amd64.Windows.msvc.idl.compiler=midl
 amd64.Windows.msvc.idl.defines=
-amd64.Windows.msvc.idl.options=
```

Comment:
Are these removals intended? If so, I think the commit message needs to explain what this is relevant to the addition of xlC_r. Same for the removal of the `x86.Windows.gpp` lines.